src/main/kotlin/org/malachite/estella/services/MailService.kt

```diff
@@ -37,6 +37,12 @@ class MailService(
     fun sendInterviewInvitationMail(offer: Offer, interview:Interview) =
         sendMail(interview.toInterviewInvitationAsMailPayload(offer))
 
+    fun sendInterviewDevInvitationMail(offer: Offer, interview:Interview, application: Application, hostMail: String) =
+        sendMail(interview.toInterviewDevInvitationAsMailPayload(offer, application,hostMail))
+
+    fun sendInterviewDateConfirmationMail(offer: Offer, interview:Interview, application: Application, mail: String) =
+        sendMail(interview.toInterviewDateConfirmationAsMailPayload(offer, application,mail))
+
```

Comment:
Do robaka nie wysyłamy maila, że ma coś ogarnąć?

Reply:
Też mi się wydaje że to jest wystarczające, można by rozważyć feature z przypominajką że ma się rozmowę wtedy lub wtedy, chociaż to by się zawierało w wysyłanym zaproszeniu w google calendar który jest jako improvment. Więc jak na moje tak bym to zostawił jak jest obecnie